playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -273,6 +273,44 @@
     include_role:
       name: lib_openshift
 
+  # Check here for bugzilla - 1463393 which will result in data loss if not handled properly
+  # We need to update the rpm before doing any node evacuations or we risk running
+  # a code path that triggers CleanOrphanPodDir which will rm -rf the data.
+  - block:
+    # rpm (non-containerized) version version
+    - block:
+      - name: Update the rpm before we proceed
+        include_role:
+          name: roles/openshift_node_upgrade
+          tasks_from: rpm_upgrade.yml
+        vars:
+          component: node
+
+      # TODO: ansible 2.4 added systemd support for daemon_reload
+      - name: reload systemd unit
+        command: systemctl daemon-reload
```

Comment:
Looks like `rpm_upgrade.yaml` also reloads systemd unit files. Do we need to this separately here?